src/components/Button/Button.tsx

```diff
@@ -32,23 +48,40 @@ class Button extends UIComponent<any, any> {
     /** Shorthand for primary content. */
     content: customPropTypes.contentShorthand,
 
+    /** A button can take the width of its container. */
+    fluid: PropTypes.bool,
+
     /** A button can be formatted to show different levels of emphasis. */
     type: PropTypes.oneOf(['primary', 'secondary']),
   }
 
-  static handledProps = ['as', 'circular', 'className', 'content', 'type']
+  public static handledProps = [
+    'as',
+    'children',
+    'circular',
+    'className',
+    'content',
+    'fluid',
+    'type',
+  ]
 
-  static defaultProps = {
+  public static defaultProps = {
     as: 'button',
   }
 
-  renderComponent({ ElementType, classes, rest }) {
+  public renderComponent({
+    ElementType,
+    classes,
+    rest,
+  }: IRenderResultConfig<IButtonProps>): ReactNode {
```

Comment:
This should already be typed by extension of the UIComponent, no?

Reply:
![image](https://user-images.githubusercontent.com/5442794/43152436-ce128b88-8f6e-11e8-9be7-fde6d407d08d.png)
doesn't look like it...